params/config.go

```diff
@@ -633,6 +602,16 @@ func (c *NodeConfig) UpdateWithMobileDefaults() {
 	if c.APIModules == "" {
 		c.APIModules = "net,web3,eth"
 	}
+
+	// Override defaultMinPoW passed by the client
+	if c.WakuConfig.Enabled {
+		c.WakuConfig.MinimumPoW = WakuMinimumPoW
+	}
+
+	if c.WhisperConfig.Enabled {
+		c.WakuConfig.MinimumPoW = WhisperMinimumPoW
```

Comment:
I'm confused, why are we setting `WakuConfig` to a `WhisperMinimumPoW`?

Reply:
ouch, that's a good reason to be confused :) well spotted